todo.txt

```diff
@@ -0,0 +1,3 @@
+TASKS TO DO:
+migrate to flex-box
```

Comment:
in fact, you almost did it :) so, I think that you can remove:
`migrate to flex-box`
`improving semantics`